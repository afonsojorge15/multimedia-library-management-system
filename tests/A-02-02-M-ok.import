BOOK:A arte de sobreviver no 36:Joao Fonseca:30:FICTION:1234567892:22
BOOK:Bairro Alto e o Budismo Zen:Zun Tse Fonseca:25:FICTION:1234567891:50
BOOK:Dicionário de Programação:Odete Fonseca:20:REFERENCE:1234567890:50
BOOK:Analise Matematica sem Mestre:Carlos Fonseca:29:SCITECH:1234567890:5
