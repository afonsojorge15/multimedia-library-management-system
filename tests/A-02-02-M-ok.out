Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Menu de obras
1 - Mostrar obra
2 - Mostrar todas as obras
3 - Efectuar pesquisa de termos
0 - Sair
Escolha uma opção: 0 - 22 de  22 - Livro - A arte de sobreviver no 36 - 30 - Ficção - Joao Fonseca - 1234567892
1 - 50 de  50 - Livro - Bairro Alto e o Budismo Zen - 25 - Ficção - Zun Tse Fonseca - 1234567891
2 - 50 de  50 - Livro - Dicionário de Programação - 20 - Referência - Odete Fonseca - 1234567890
3 - 5 de  5 - Livro - Analise Matematica sem Mestre - 29 - Técnica e Científica - Carlos Fonseca - 1234567890
Menu de obras
1 - Mostrar obra
2 - Mostrar todas as obras
3 - Efectuar pesquisa de termos
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 
